### ISSUE TITLE
If user skips ahead a section and then scrolls up, show them last slide of previous section by default.

### ISSUE THREAD

**Issue by etlingfs** [alvarotrigo/fullPage.js#2499]
### Description

I believe this is a request for a new feature, or an enhancement, rather than a bug.

If a user skips ahead to a section via an anchor link, and then scrolls back UP to a previous section, I would like to be able to transition directly to the last slide of the section rather than the first.

### Link to isolated reproduction with no external CSS / JS

Here is an example built specifically to demonstrate: http://willetling.com/tests/fullpage_scrolling/

### Steps to follow on the example url:
1. After page loads, click **Section C** in the top navigation bar
2. After **Section C, Slide 1** has loaded, scroll up to go to **Section B**
3. B.1 will appear, because the user has never viewed Section B before so of course it is on B.1. However, since B.3 is technically the preceding slide to C.1, I would like the scrolling action to reveal B.3. 

Does that make sense? Let me know what else I can do to explain. Thank you.

**Comment by alvarotrigo** [comment]
Yeah I see what you mean now. But this feature would only make sense when using the scroll horizontally extension. 

I'll mark it as a possible enhancement.

**Comment by etlingfs** [comment]
@alvarotrigo - thanks so much. It certainly only makes sense when using Scroll Horizontally. I appreciate you considering it.

Also I realized last night that if I really want this "true narrative" behavior, maybe I could just put my content into many slides within a single section, rather than multiple sections with several slides. There's something very nice, however, about having "chapters" of content.
